hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowDataParquetWriteSupport.java

```diff
@@ -71,7 +70,7 @@ public HoodieBloomFilterRowDataWriteSupport(BloomFilter bloomFilter) {
 
     @Override
     protected byte[] getUTF8Bytes(String key) {
-      return key.getBytes(StandardCharsets.UTF_8);
+      return getUTF8Bytes(key);
```

Comment:
Need to import `StringUtils.getUTF8Bytes` otherwise it's recursive. i'll push a fix.